zach_kaz/funcinter2.txt

```diff
@@ -0,0 +1,37 @@
+x = [ [5,2,3], [10,8,9] ] 
+students = [
+     {'first_name':  'Michael', 'last_name' : 'Jordan'},
+     {'first_name' : 'John', 'last_name' : 'Rosales'}
+]
+sports_directory = {
+    'basketball' : ['Kobe', 'Jordan', 'James', 'Curry'],
+    'soccer' : ['Messi', 'Ronaldo', 'Rooney']
+}
+z = [ {'x': 10, 'y': 20} ]
+
+x[1[0]]=15
+students[0.last_name]='Bryant'
```

Comment:
dictionary inside an array //elements of the array are dictionaries
students[0] is {'first_name':  'Michael', 'last_name' : 'Jordan'}
students[0]['lsat_name'] is "Jordan"
print things at each level. It will help you understand